air/src/execution_step/execution_context/stream_maps_variables/stream_map_key.rs

```diff
@@ -14,42 +14,70 @@
  * limitations under the License.
  */
 
-use crate::execution_step::execution_context::stream_maps_variables::errors::unsupported_map_key_type;
 use crate::execution_step::ValueAggregate;
-use crate::CatchableError;
-use crate::ExecutionError;
 use crate::JValue;
 
 use serde::Serialize;
 use std::borrow::Cow;
+use std::fmt::Display;
+use std::fmt::Formatter;
 
-#[derive(Clone, PartialEq, Eq, Hash)]
+pub(crate) static KEY_FIELD_NAME: &str = "key";
+
+#[derive(Clone, PartialEq, Debug, Eq, Hash)]
 pub(crate) enum StreamMapKey<'value> {
```

Comment:
In several places we have an implicit convertation of integer into a string for keys, in some places there are some comments why it could be unexpected and even dangerous. I think that the more proper solution would be introducing a dedicated trait for that. But let's do it in a separate PR and now only left a TODO comment about that somewhere in this file.

Reply:
I suggest you to share some details in future b/c I don't know what to tell exactly and how would a dedicated trait facilitates an overlap of integer and string values.
Added a TODO note blindly though.